hdl21/sim/tests/test_sim.py

```diff
@@ -315,17 +315,35 @@ def test_empty_sim2():
     assert not len(r.an)  # No analysis inputs, no analysis results
 
 
-@pytest.mark.skipif(
-    vlsirtools.spice.default() is None,
-    reason="No simulator available",
-)
-def test_sim_async_caller():
-    """# Test invoking simulation from an async caller"""
+def test_multi_sim():
+    """Test multiple Sims in parallel"""
+    s1 = Sim(tb=empty_tb(1), attrs=[])
+    s2 = Sim(tb=empty_tb(2), attrs=[])
+    s3 = Sim(tb=empty_tb(3), attrs=[])
+    s4 = Sim(tb=empty_tb(4), attrs=[])
+
+    r = sim(to_proto([s1, s2, s3, s4]), SimOptions(fmt=ResultFormat.VLSIR_PROTO))
+
+    for a in r:
+        assert isinstance(a, vsp.SimResult)
+        assert not len(a.an)
+
+
+def really_empty_tb() -> h.Module:
+    @h.module
+    class ReallyEmptyTb:
+        """An Empty TestBench, this time REALLY empty.
+        For inducing an error in a multi-sim below."""
+
+    return ReallyEmptyTb
+
 
-    async def caller():
-        """# The asynchronous caller of `sim_async`"""
-        s = Sim(tb=empty_tb(), attrs=[])
-        return await s.run_async(SimOptions(fmt=ResultFormat.SIM_DATA))
+def test_multi_sim_error():
+    """Test multiple sims fail to avoid erdewit/nest_asyncio#57"""
```

Comment:
Not sure I follow the deal here?  
We got rid of `nest_asyncio`, right?  

The idea of the test looks good - several `Sims` go in, one fails, the others don't, and the result is an `Exception`. So maybe I just get confused by the "fail to avoid (that issue)" comment.

Reply:
Yeah, it's badly worded, "Test that a failure in multiple concurrent sims fails, unlike in erdewit/nest_asyncio#57", will be better.